lib/assets/javascripts/turbograft/link.js

```diff
@@ -0,0 +1,67 @@
+(function() {
+  var extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
+    hasProp = {}.hasOwnProperty,
+    slice = [].slice;
+
+  /* The Link class derives from the ComponentUrl class, but is built from an
+   * existing link element.  Provides verification functionality for Turbolinks
+   * to use in determining whether it should process the link when clicked.
+   */
+  window.Link = (function(superClass) {
+    extend(Link, superClass);
+
+    Link.HTML_EXTENSIONS = ['html'];
+
+    Link.allowExtensions = function() {
+      var extension, extensions, i, len;
+      extensions = 1 <= arguments.length ? slice.call(arguments, 0) : [];
+      for (i = 0, len = extensions.length; i < len; i++) {
+        extension = extensions[i];
+        Link.HTML_EXTENSIONS.push(extension);
+      }
+      return Link.HTML_EXTENSIONS;
+    };
+
+    function Link(link) {
+      if (link.constructor === Link) {
+        return link;
+      }
+      Link.__super__.constructor.call(this, link.href, link);
```

Comment:
Ok, I figured out that this line needs to be changed to make all the `class` syntax to work:
```suggestion
      super(link.href, link);
```
Otherwise, JS complains that a constructor was not called with `new`.